jerry-core/include/jerryscript-core.h

```diff
@@ -356,6 +356,11 @@ bool jerry_value_is_promise (const jerry_value_t value);
 bool jerry_value_is_string (const jerry_value_t value);
 bool jerry_value_is_undefined (const jerry_value_t value);
 
+/*
+ * Comparison functions
+ */
+jerry_value_t jerry_strict_equal (const jerry_value_t lhs, const jerry_value_t rhs);
```

Comment:
If strict comparison is made part of the public API, shouldn't `ecma_op_abstract_equality_compare` have its API counterpart, too?

Reply:
Hm, I've missed @galpeter 's suggestion. I'm not sure I completely agree with the one function in this PR though. The < and > operators aren't necessarily needed here, but === and == are quite related, so publishing them via API should happen together, IMO.

Reply:
@akosthekiss To have other comparison api functions was suggested in my post above, but I had another "idea" which was not written down yet: As we introduce more and more api functions for comparisons, does it make sense to have a single function for each of them or is it possible to have a single api function which can be configured to perform different kind of comparisons?

For example something like this (do not take the naming into account just the concept):
```c
enum jerry_binary_operation {
    JERRY_OP_STRICT_EQUALS, // lhs === rhs
    JERRY_OP_EQUALS, // lhs == rhs
    JERRY_OP_NOT_EQUAL, // lhs != rhs
    JERRY_OP_LESS,  // lhs < rhs
    ...
    JERRY_OP_ADD, // lhs + rhs
};

jerry_value_t
jerry_op (jerry_binary_operation op, jerry_value_t lhs, jerry_value_t rhs);
```

I'm not sure which approach is better: having api functions for each operation or having a single one. (Maybe this discussion needs its own issue.)

Reply:
I like this idea.

Reply:
@galpeter I like your idea, I will do it.